packages/design-system/src/styles/components/_Accordion.scss

```diff
@@ -41,13 +41,32 @@ $accordion-header-font-family: $font-sans;
   text-align: left;
   text-decoration: none;
   width: 100%;
+
   &:hover {
     background-color: $accordion-header-bg-hover-color;
   }
+
   &:focus {
     @include focus-styles;
   }
 
+  /* stylelint-disable */
+  @media (-ms-high-contrast: active), (forced-colors: active) {
+    -ms-high-contrast-adjust: none;
+    background-color: LinkText;
+    color: window;
+
+    &:hover,
+    &:focus {
+      -ms-high-contrast-adjust: none;
+      background-color: window;
+      color: LinkText;
+      outline-offset: -4px;
+      outline: 4px solid LinkText;
```

Comment:
Should we consider using our spacer variable here for the 4px?

Reply:
Oh, good call. Added!